examples/airflow_local/run_schedule.py

```diff
@@ -0,0 +1,121 @@
+#  Copyright (c) ZenML GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+from datetime import datetime, timedelta
```

Comment:
You could use one of two tricks to avoid duplicating the whole example code:

* move the common elements (steps, pipelines, utility functions and so on) to an `example.py` or `pipelines.py` file and import everything from it into `run.py` and `run_schedule.py`. This approach is used in the [mlflow_tracking example](https://github.com/zenml-io/zenml/tree/main/examples/mlflow_tracking) 
* use click to add a command line argument to `run.py` that controls which type of pipeline run is launched (scheduled vs. one-time)

Reply:
Good point. Done.